docs/domains/sparse_linear_algebra.rst

```diff
@@ -38,3 +38,145 @@ Currently known limitations:
   ``oneapi::mkl::unimplemented`` exception.
 - Scalar parameters ``alpha`` and ``beta`` should be host pointers to prevent
   synchronizations and copies to the host.
+
+
+cuSPARSE backend
+----------------
+
+Currently known limitations:
+
+- The COO format requires the indices to be sorted by row. See the `cuSPARSE
+  documentation
+  <https://docs.nvidia.com/cuda/cusparse/index.html#coordinate-coo>`_. Sparse
+  operations using matrices with the COO format without the property
+  ``matrix_property::sorted_by_rows`` or ``matrix_property::sorted`` will throw
+  an ``oneapi::mkl::unimplemented`` exception.
+- Using ``spmm`` with the algorithm ``spmm_alg::csr_alg3`` and an ``opA`` other
+  than ``transpose::nontrans`` or an ``opB`` ``transpose::conjtrans`` will throw
+  an ``oneapi::mkl::unimplemented`` exception.
+- Using ``spmv`` with a ``type_view`` other than ``matrix_descr::general`` will
+  throw an ``oneapi::mkl::unimplemented`` exception.
+
+
+Operation algorithms mapping
+----------------------------
+
+The following tables describe how a oneMKL SYCL Interface algorithm maps to the
+backend's algorithms. Refer to the backend's documentation for a more detailed
+explanation of the algorithms.
+
+Backends with no equivalent algorithms will fallback to the backend's default
+behavior.
+
+
+spmm
+^^^^
+
+.. list-table::
+   :header-rows: 1
+   :widths: 10 30 45
+
+   * - Value
+     - Description
+     - Backend equivalent
+   * - ``default_optimize_alg``
+     - Default algorithm.
+     - | MKL: none
+       | cuSPARSE: ``CUSPARSE_SPMM_ALG_DEFAULT``
+   * - ``no_optimize_alg``
+     - Default algorithm but may skip some optimizations. Useful only if an
+       operation with the same configuration is run once.
```

Comment:
```suggestion
     - Default algorithm but may skip some optimizations. Typically useful only
       if an operation with the same configuration is run once.
```

Reply:
This is now removed in https://github.com/oneapi-src/oneMKL/pull/527/commits/f73fd0b0df04205a71c1e71eeaddd316f7bc4464